ci/common.sh

```diff
@@ -5,6 +5,7 @@ es_port=${ES_PORT:-80}
 
 default_operator_image="quay.io/benchmark-operator/benchmark-operator:master"
 
+default_ripsaw_image_spec="quay.io/cloud-bulldozer"
```

Comment:
just a small nit, this is not a spec, but an org.

Reply:
Replaced by default_ripsaw_image_prefix